rfcs/20221124-monorepo.md

```diff
@@ -248,7 +280,29 @@ To clean up the workflows we will define three new actions. The code for all thr
 
 #### Update workflows
 
-With this done, the development on the API and frontend can continue inside their subdirectories. The development of both parts will be independent. At least until we reach [long-term consolidation](#step-5-long-term-consolidation).
+Workflows with major changes:
+
+- `ci_cd.yml` from the API will absorb `ci.yml` from the frontend
+- `lint.yml` will be deleted
+
+Updates:
+
+- `migration_safety_warning.yml`
+- `generate_pot.yml`
+
+With this done, the development on the API and frontend can continue inside their subdirectories. The development of both parts will be independent, at least until we reach [long-term consolidation](#step-5-long-term-consolidation).
+
+#### Deployment
+
+##### Staging
+
+The soon-to-be-ECS-based API and ECS-based frontend will continue to deploy to staging via the CI + CD pipeline, with deployment starting as soon as all CI checks have passed. They will use similar code as the frontend auto-deploy for staging used currently.
+
+These will be separate jobs with specific [path-based filters](https://github.com/dorny/paths-filter).
+
+##### Production
+
+For production, we will not be able to use GitHub Releases and will have to use a manually-triggered workflow to build the assets and tag them. The tag can be set via a workflow input (simple) or can be calculated based on the update scope of major, minor or patch (not as simple).
```

Comment:
Another way to simplify this is for us to switch to SHA based tags for release deployment. Rather than creating a new build, just use the build tagged with the `HEAD` SHA.

Reply:
The slight issue with SHA tags would be that it's harder to identify versioning with them. Given two hashes, it's not possible to say which is the newer version.

Reply:
Ah, good point!